lib/clients/graphql/graphql_client.ts

```diff
@@ -79,6 +79,7 @@ export class GraphqlClient {
       throw new ShopifyErrors.GraphqlQueryError({
         message: 'GraphQL query returned errors',
         response: result.body as unknown as {[key: string]: unknown},
+        headers: result.headers,
```

Comment:
I wonder if the idiomatic expectation would be that `GraphqlQueryError` contains a `response` property that itself then contains a `body` property and a `headers` property (in alignment with a Node `response` object).

Reply:
Let's leave your change as-is.  Setting `response` to contain `body` and `headers` would become an unnecessary breaking change, I think.